### ISSUE TITLE
Empty Decks and At-Start Stacks should allow re-positioning

### ISSUE THREAD

**Issue by BrentEaston** [vassalengine/vassal#12509]
Empty Decks that have a Deck GKC or other child allow re-positioning via a white square. Completely empty Decks and At-start stacks should have the same functionality.

**Comment by riverwanderer** [comment]
I think there's an issue with the empty deck feature; tested using the Study in Emerald module (as "Red" player").

Discard Deck is empty, but the position is shown by the black rectangle on the map/background. Cards do snap to that position when dragged. However, the white-marker is offset. Is this a side-effect of your 100,100 change ?

<img width="292" alt="image" src="https://github.com/vassalengine/vassal/assets/58135975/1b704bc6-5f1e-4b61-8cd8-b2c3f2e6690e">

The Draw Deck on the same hand window is pre-populated and performs as expected. As does the Discard Pile if you add an initial card to it.

**Comment by BrentEaston** [comment]
Hmm, That should only happen in a newly created Deck, but that looks a bit suspicious. Thanks.

But I don't see that. Exactly what actions did you do to get there

![image](https://github.com/vassalengine/vassal/assets/1622667/de8b4692-5d18-411c-96ae-7759970cf348)

**Comment by riverwanderer** [comment]
Re-tested using C&C Medieval module, unmodified (v1.8). Same result as I found before - the empty stack indictor is offset.
Also, if I change the setting of "show other stacks / decks", the arrow keys no longer adjust the position of the deck (don't know if this is an old bug or to do with the change, but I didn't notice this problem when I used the feature a while back).

To replicate; edit the C&C Medieval module. Find the Discard Pile:
<img width="286" alt="image" src="https://github.com/vassalengine/vassal/assets/58135975/081d5f5c-1337-45c8-9f78-536909d3a5ee">

Check Discard Pile properties and click the Reposition button and check "show other stacks / decks". Notice how the empty deck marker overlays the Draw Deck to its right, it is also dropped down slightly. The actual configuration is on the same Y value and does not overlap. The coordinates shown are correct (see below).

Notice also that the arrow keys do not work - assuming that you had to alter the checkbox setting. If you cancel and re-try, the arrow keys will now work. This is regardless of checkbox ticked or not - the trigger is when the checkbox is changed.

<img width="978" alt="image" src="https://github.com/vassalengine/vassal/assets/58135975/3e090b8e-5937-4153-a476-9dcfefe8b74e">
